linux/system/autoupdates.sls

```diff
@@ -0,0 +1,24 @@
+{%- from "linux/map.jinja" import system with context %}
+{%- if system.enabled %}
+
+{%- if system.autoupdates is defined %}
+
+{%- if system.autoupdates.pkgs > 0 %}
```

Comment:
After checking, it was still working without the filter, but I think I can remove the `{%- if system.autoupdates.pkgs > 0 %}` statement altogether